src/compiler/transformers/es6.ts

```diff
@@ -1217,7 +1217,10 @@ namespace ts {
             const body = node.body;
             if (isBlock(body)) {
                 statementsLocation = body.statements;
-                addRange(statements, visitNodes(body.statements, visitor, isStatement));
+                // ensureUseStrict is false because no new prologue-directive should be added.
+                // the function will simply put already existed directive as a first statement in the target statement-array.

```

Comment:
I would word this like so:
"addPrologueDirectives will simply put already-existing directives at the beginning of the target statement-array."